src/operator/tensor/ordering_op.cc

```diff
@@ -114,7 +114,7 @@ Examples::
              [ 1.,  3.]]
 
   // flattens and then sorts
-  sort(x) = [ 1.,  1.,  3.,  4.]
+  sort(x, axis=None) = [ 1.,  1.,  3.,  4.]
```

Comment:
This sort makes sense. 
But I'm thrown off by the other example. Sorting along the last axis would be 3 then 4 unless it is descending rather than ascending. Does this need to be specified for clarity?

Reply:
Hi, 
Thanks for your reply! 

Do you mean this example?
// sorts along the last axis
`sort(x) = [[ 1.,  4.],
           [ 1.,  3.]]`

Since x = `[[ 1, 4],[ 3, 1]]`, the pairs in the last axis are `[1,4]` and `[3,1]`. Sorting only along the last axis in a descend way would be `[1,4]` and `[1,3]`. So, in my opinion, it is correct? or maybe I'm wrong?

Reply:
I tested it in numpy and it matches fine.

Reply:
Sorry that I didn't get your point. What I did wrong in this PR? 

If the argument `axis=None` is not given, this function will sort the array alone the last axis. However, in the second example, we want the array to be flattened and then sorted.

Reply:
Everything is fine. I might have needed more examples to see the patterns, but I understand that if you're conversant with NumPy then this all works as expected.